attributes/config.rb

```diff
@@ -3,8 +3,12 @@
 # Config defaults options
 default['stackstorm']['config'] = {
   api_url: 'http://localhost:9101',
-  conf_root: '/etc',
+  api_host: '0.0.0.0',
```

Comment:
@lakshmi-kannan That thing comes from the https://github.com/StackStorm/st2/blob/master/conf/st2.package.conf#L5

I think we should edit `st2.package.conf` and `st2.prod.conf` first, if it's new recommended value.

And Chef recipe will follow that.